src/style.css

```diff
@@ -1,3 +1,66 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header-link {
+  height: 40px;
+  width: 40px;
+}
+
+.header-nav {
+  height: 60px;
```

Comment:
the height should be set for nav links (not the header), take it from the design.